pkg/storaged/iscsi-panel.jsx

```diff
@@ -144,7 +140,7 @@
                                     if (err.message.indexOf("authorization") != -1)
                                         iscsi_add_with_creds(discover_vals, vals);
                                     else
-                                        return cockpit.reject(err);
+                                        return Promise.reject(err);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)